### PR TITLE
WINDUPRULE-137  Catchall rules should have Optional severity

### PR DESCRIPTION
**Pull request by OndraZizka**
I'm giving them "Potential" severity, as it seems that's what the user really wants. Correct me if I am wrong.

### CODE REVIEW

File: rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml [hunk 1]
```diff
@@ -30,10 +30,10 @@
                         </not>
                     </when>
                     <perform>
-                        <classification title="Doug Lea Concurrency util" severity="mandatory">
+                        <classification title="Doug Lea Concurrency util" severity="potential">
```
Comment (jsight) [inline rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml]: IMO, this one should just be optional. I can't imagine a reason for it to be mandatory for an EAP migration.

**Comment by jsight** [comment]
I agree. Can we mention you at the meeting to make sure we are all on the same page?

**Comment by jsight** [comment]
This just needs a rebase, I think.

File: rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml (hunk 1 above)
Comment (OndraZizka) [inline rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml]: Optional and not potential? Potential is more than optional? And does "potential" imply that it's a mandatory change? I'm a bit confused.

**Comment by OndraZizka** [comment]
Rebased.

File: rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml (hunk 1 above)
Comment (jsight) [inline rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml]: Well, it is really weird for this to be a catchall rule to begin with, IMO.

Comment (mareknovotny) [inline rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml]: Frankly I am not against to recategorized it. Just for a reminding, these all catch rules was to mimic the Windup 0.7 feature which identified some general packages.

**Pull request merged**